### ISSUE TITLE
JSON API responses are no longer reformatted under Pretty view

### ISSUE THREAD

**Issue by erhhung** [a85/POSTMan-Chrome-Extension#9908]
**Describe the Issue**

API response with Content-Type "`application/json`" is **no longer** being reformatted under **Pretty** view.

If there's an optimization where large, compacted, JSON responses are not formatted automatically for performance reasons, then there should be a button that lets the user format it if so desired.

**Steps to Reproduce**

1. I upgraded to the latest Postman Desktop client, version 8.0.8 on macOS.
2. Ran the same API request (multiple times) as I had prior to restarting the app for upgrade.
3. API response Content-Type is "`application/json`"; pretty view is no longer formatting it—it remains compacted.
4. The response is 1 MB long, so I thought it's an optimization for large responses, but I see it for tiny responses as well.

**Environment Information**

 - Platform Type: Desktop
 - Postman Version: 8.0.8
 - Operating System: macOS 10.15.7

**Comment by DannyDainton** [comment]
Hey @erhhung

I know this sounds like a really obvious thing to do but can you fully quit the app and restart it again.

We have had folks in the past report issues with JSON formating and that did the trick.

Just want to suggest the easy stuff first 😁

**Comment by erhhung** [comment]
The app upgrade behavior on macOS is a little weird. I originally reported this issue right after I was prompted to upgrade and restart, which I complied. I saw and reported the issue for v8.0.8.

I just shut down the app and relaunched it, and the app popped up a notice that I'm now running 8.2.1. I wasn't previously being prompted about an available update, though—it just updated on its own?
I ran the same requests, and it was still not auto-formatting, so I shut down the app again (this in the span of a minute tops).

The app relaunched and now it tells me I've upgraded to 8.3.1. Is there some sort of frequent updates that are queued? Why didn't it just upgrade me to the latest a minute ago?
So I decided to quit the app and keep relaunching until it doesn't say I've upgraded again.
I tried the request in 8.3.1, and, now, finally, it's prettifying the responses in both the response pane and in the Postman Console, which wasn't showing anything at all before.

So this issue has now resolved, although the upgrade process is a bit wonky.
